yw_website/yw_website/apps/website/templates/includeable/tagarray.html

```diff
@@ -4,11 +4,15 @@
 
 <div class="tags">
 {% for tag in tags %}
+{% if can_edit_tags %}
   <div class="control">
     <div class="tags has-addons">
+      {% endif %}
       <a href="{% url 'home' %}?q={{ tag.tag.title }}" class="tag is-link">{{tag.tag.title}}</a>
+      {% if can_edit_tags %}
       <a href="{% url 'tag_delete' tag.tag.id %}" class="tag is-delete"></a>
```

Comment:
not sure why we need this two if statements, they don't really make sense. All you should need is 
```
{% if can_edit_tags %}
      <a href="{% url 'tag_delete' tag.tag.id %}" class="tag is-delete"></a>
{% endif  %}
```

Reply:
it's because of the nested container css struct

Reply:
do you mean that the `has-addons` class is the class that puts the X next to the tag?

Intuitively it would seem that the `is-delete` class would do that?

Reply:
has-addons combines them so it looks like the same tag with an x button on it

Reply:
since the other class is the hyperlinked part

Reply:
does it look weird the way I suggested? just curious now

Reply:
yea looks like all the tags are glued together